cegs_portal/search/views/v1/dna_features.py

```diff
@@ -75,6 +86,15 @@ def get(self, request, options, data, id_type, feature_id):
 
             feature_reos.append((feature, sources, targets))
 
+        non_targeting_reos = []
+        for feature in data.all():
+            reos = DNAFeatureSearch.non_targeting_reo_search(feature, options.get("sig_only"))
+            if reos.exists():
+                non_targeting_reos.extend(list(reos))
```

Comment:
Getting the actual list of REOs should done in JS via an as-yet-to-be-implemented endpoint (also part of this task). See how source_reo_search and target_reo_search above are done.